### PR TITLE
Changes to the Landing and the About Pages

### PR DESCRIPTION
**Pull request by martha-thomae**
Now the landing page points to the 'about' page after its first paragraph to be able to read more about MEI. The 'about' page opens with the exact same paragraph contained in the landing page, but it (1) includes links for the specs and guidelines and (2) expands into other information.

This change was judged necessary since both pages contained pretty similar information, they seemed redundant (same information with a few variations here and there in the wording). Now they have the exact same text, but expanding more in the 'about'. 

It was also pointed out during the workshop that someone reading this paragraph would wonder about how to find the schema and guidelines, so links were included in the About page.

Also included a link to GitHub in the landing page inviting people to collaborate through issues and PRs. 

Smaller changes:
- 'representation' changed into 'encoding system': this change was to avoid repetition, soon after the 'representation'  we found the words 'represent' (see bullet points)
- changed reference to mailing list because when clicking on the link we are in the community contacts page, where there are various options for getting in contact with the community (not only through the mailing list)

These were the changes discussed with Esperanza Rodriguez during the workshop. Please let me know if other changes are deemed necessary.

### CODE REVIEW

**Review by musicEnfanthen — COMMENTED** [review]
Thanks @martha-thomae and Esperanza. Please see comments.

File: about/index.md [hunk 1]
```diff
@@ -4,17 +4,14 @@ title: "What is MEI?"
...
-
+The Music Encoding Initiative (MEI) is a community-driven, open-source effort to define a system for encoding musical documents in a machine-readable structure. MEI brings together specialists from various music research communities, including technologists, librarians, historians, and theorists in a common effort to define best practices for representing a broad range of musical documents and structures. The results of these discussions are formalized in the [MEI schema](/resources/schemas.html), a core set of rules for recording physical and intellectual characteristics of music notation documents expressed as an eXtensible Markup Language ([XML](https://web.archive.org/web/20191028132600/https://www.tei-c.org/release/doc/tei-p5-doc/en/html/SG.html)) schema. It is complemented by the [MEI Guidelines](https://music-encoding.org/guidelines/v4/content/), which provide detailed explanations of the components of the MEI model and best practices suggestions. The schema is developed and maintained by the [MEI Technical Team](/community/technical-team.html).
```
[the remark this answers is not in the document]
Comment (martha-thomae) [reply about/index.md]: True, it wasn't necessary to remove it. I added it back.

**Comment by martha-thomae** [comment]
All changes done. Thanks, @musicEnfanthen!

**Comment by musicEnfanthen** [comment]
Thanks @martha-thomae and Esperanza

**Pull request merged**